packages/graphql-mocks/src/graphql/utils/build-context.ts

```diff
@@ -11,9 +11,15 @@ export function buildContext({
   queryContext?: GraphQLArgs['contextValue'];
   packOptions: PackOptions;
 }): ResolverContext {
-  return {
+  const context = {
     ...(initialContext as Record<string, unknown>),
     ...(queryContext as Record<string, unknown>),
-    pack: packOptions,
   };
+
+  Object.defineProperty(context, 'pack', {
```

Comment:
Better way of ensuring that `pack` isn't tampered with on context between middleware calls than re-embedding it

Reply:
Done in #275